File: testing-copilot/testing.R
calculateDays() {
  echo "days($1, $2, $3)" | R --no-save
}
calculateHours() {
  echo "hours($1, $2, $3)" | R --no-save
}
calculateMinutes() {
  echo "minutes($1, $2, $3)" | R --no-save
}
calculateSeconds() {
  echo "seconds($1, $2, $3)" | R --no-save
}
calculateTime() {
  echo "time($1, $2, $3)" | R --no-save
}
calculateTimeFromDays() {
  echo "timeFromDays($1, $2, $3)" | R --no-save
}

File: 6_Programming_Languages/language-examples/testing.py
calculateDays = lambda x: x * 365
calculateHours = lambda x: x * 365 * 24
calculateMinutes = lambda x: x * 365 * 24 * 60
calculateSeconds = lambda x: x * 365 * 24 * 60 * 60

calculateTime = lambda x, y: calculateDays(x) + calculateHours(y)